fix(复制): 选中复制

heyumoAi opened the pull request:
Description: 复制字符位置计算错误

Log: https://pms.uniontech.com/bug-view-153531.html

worldofpeace commented:
You need to do conditional compilation to have this build with older versions.
This commit in elementary-terminal would be helpful https://github.com/elementary/terminal/commit/7dd6f93c9cd9e3d44407649a99ab0c3c17670a1a.
This LGTM.

sgn commented:
I separated into 2 patches series.
I tried to build with vte3 0.59.{0,91,92} and 0.60.{0,1}

Commit:
fix(复制): 选中复制

Description: 复制字符位置计算错误

Log: bug：https://pms.uniontech.com/bug-view-153531.html
